react/features/settings/components/native/SettingsView.tsx

```diff
@@ -166,14 +209,40 @@ class SettingsView extends AbstractSettingsView<Props, State> {
 
         // Bind event handlers so they are only bound once per instance.
         this._onBlurServerURL = this._onBlurServerURL.bind(this);
+        this._onChangeDisplayName = this._onChangeDisplayName.bind(this);
+        this._onChangeEmail = this._onChangeEmail.bind(this);
+        this._onChangeServerURL = this._onChangeServerURL.bind(this);
         this._onClose = this._onClose.bind(this);
         this._onDisableCallIntegration = this._onDisableCallIntegration.bind(this);
         this._onDisableCrashReporting = this._onDisableCrashReporting.bind(this);
         this._onDisableP2P = this._onDisableP2P.bind(this);
+        this._onDisableSelfView = this._onDisableSelfView.bind(this);
+        this._onStartAudioMutedChange
+            = this._onStartAudioMutedChange.bind(this);
+        this._onStartReactionsMutedChange
+            = this._onStartReactionsMutedChange.bind(this);
+        this._onStartVideoMutedChange
+            = this._onStartVideoMutedChange.bind(this);
         this._setURLFieldReference = this._setURLFieldReference.bind(this);
         this._showURLAlert = this._showURLAlert.bind(this);
     }
 
+    /**
+     * Sets a timeout to hide the toolbar when the toolbar is shown.
+     *
+     * @inheritdoc
+     * @returns {void}
+     */
+    componentDidUpdate(prevProps: Props) {
+        const { settings } = this.props;
+
+        if (prevProps.settings !== settings) {
```

Comment:
Since this is an object, you need to do a deep equals comparison, you can use lodash isEqual.